WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Reader Messages/Presented Views/PointOfSaleCardPresentPaymentCaptureFailedView.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+
+struct PointOfSaleCardPresentPaymentCaptureFailedView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    var body: some View {
+        VStack {
+            Text(Localization.title)
+
+            Image(uiImage: .paymentErrorImage)
+
+            Text(Localization.errorDetails)
+
+            Button(Localization.cancel,
+                   action: {
+                dismiss()
+            })
+            .buttonStyle(SecondaryButtonStyle())
+        }
+    }
+}
+
+private extension PointOfSaleCardPresentPaymentCaptureFailedView {
+    enum Localization {
+        static let title = NSLocalizedString(
+            "pointOfSale.cardPresentPayment.alert.paymentCaptureError.title",
+            value: "Please check order payment status",
+            comment: "Title of the alert presented when payment capture fails."
+        )
+
+        static let errorDetails = NSLocalizedString(
+            "pointOfSale.cardPresentPayment.alert.paymentCaptureError.errorDetails",
+            value: "Due to an error from capturing payment and refreshing order, we couldn't load complete order information. " +
+            "To avoid undercharging or double charging, please check the latest order separately before proceeding.",
+            comment: "Subtitle of the alert presented when payment capture fails."
+        )
+
+        static let cancel = NSLocalizedString(
```

Comment:
Should we maybe rename this from `cancel` to `dismiss` or `understand` to be closer to translation or the function called when button is tapped?

Reply:
Sure, updated in https://github.com/woocommerce/woocommerce-ios/pull/13215/commits/9934309a40e2171f5b0603aae28f3dd1d9fc2f99